setup.py

```diff
@@ -43,7 +43,8 @@ def main():
         author='holger krekel',
         author_email='holger@merlinux.eu',
         packages=['tox'],
-        entry_points={'console_scripts': 'tox=tox:cmdline\ntox-quickstart=tox._quickstart:main'},
+        entry_points={'console_scripts': ['tox=tox:cmdline',
+                                          'tox-quickstart=tox._quickstart:main']},
```

Comment:
cool, I was wondering why this was not done this way originally, and assumed it was some sort of a backward compatibility hack